cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -43,6 +43,7 @@ var (
 	selectEndDeviceFlags     = &pflag.FlagSet{}
 	setEndDeviceFlags        = &pflag.FlagSet{}
 	endDeviceFlattenPaths    = []string{"provisioning_data"}
+	pictureFlags             = &pflag.FlagSet{}
```

Comment:
`pictureFlags` lives in the `commands` package, so if you only use it for end device pictures, you need to call it `endDevicePictureFlags`.